benchmarks/atomic/main.cpp

```diff
@@ -1,124 +1,120 @@
+      printf("  D:   Distance from index i to do atomics into (randomly)\n");
+      printf("  K:   Number of FMAD per atomic\n");
+      printf("  R:   Number of repeats of the experiments\n");
+      printf("  T:   Type of atomic\n");
+      printf("       1 - int\n");
+      printf("       2 - long\n");
+      printf("       3 - float\n");
+      printf("       4 - double\n");
+      printf("       5 - complex<double>\n");
+      printf("Example Input GPU:\n");
+      printf("  Histogram : 1000000 1000 1 1000 1 10 1\n");
+      printf("  MD Force : 100000 100000 100 1000 20 10 4\n");
+      printf("  Matrix Assembly : 100000 1000000 50 1000 20 10 4\n");
+      Kokkos::finalize();
+      return 0;
+    }
 
+    int L    = atoi(argv[1]);
+    int N    = atoi(argv[2]);
+    int M    = atoi(argv[3]);
+    int D    = atoi(argv[4]);
+    int K    = atoi(argv[5]);
+    int R    = atoi(argv[6]);
+    int type = atoi(argv[7]);
 
-  int L = atoi(argv[1]);
-  int N = atoi(argv[2]);
-  int M = atoi(argv[3]);
-  int D = atoi(argv[4]); 
-  int K = atoi(argv[5]);
-  int R = atoi(argv[6]); 
-  int type = atoi(argv[7]);
- 
-  Kokkos::View<int*> offsets("Offsets",L,M);
-  Kokkos::Random_XorShift64_Pool<> pool(12371);
-  Kokkos::fill_random(offsets,pool,D);
-  double time = 0;
-  if(type==1)
-    time  = test_atomic<int>(L,N,M,K,R,offsets);
-  if(type==2)
-    time = test_atomic<long>(L,N,M,K,R,offsets);
-  if(type==3)
-    time = test_atomic<float>(L,N,M,K,R,offsets);
-  if(type==4)
-    time = test_atomic<double>(L,N,M,K,R,offsets);
-  if(type==5)
-    time = test_atomic<Kokkos::complex<double> >(L,N,M,K,R,offsets);
+    Kokkos::View<int*> offsets("Offsets", L, M);
+    Kokkos::Random_XorShift64_Pool<> pool(12371);
+    Kokkos::fill_random(offsets, pool, D);
+    double time = 0;
+    if (type == 1) time = test_atomic<int>(L, N, M, K, R, offsets);
+    if (type == 2) time = test_atomic<long>(L, N, M, K, R, offsets);
+    if (type == 3) time = test_atomic<float>(L, N, M, K, R, offsets);
+    if (type == 4) time = test_atomic<double>(L, N, M, K, R, offsets);
+    if (type == 5)
+      time = test_atomic<Kokkos::complex<double> >(L, N, M, K, R, offsets);
 
-  double time2 = 1;
-  if(type==1)
-    time2 = test_no_atomic<int>(L,N,M,K,R,offsets);
-  if(type==2)
-    time2 = test_no_atomic<long>(L,N,M,K,R,offsets);
-  if(type==3)
-    time2 = test_no_atomic<float>(L,N,M,K,R,offsets);
-  if(type==4)
-    time2 = test_no_atomic<double>(L,N,M,K,R,offsets);
-  if(type==5)
-    time2 = test_no_atomic<Kokkos::complex<double> >(L,N,M,K,R,offsets);
+    double time2 = 1;
+    if (type == 1) time2 = test_no_atomic<int>(L, N, M, K, R, offsets);
+    if (type == 2) time2 = test_no_atomic<long>(L, N, M, K, R, offsets);
+    if (type == 3) time2 = test_no_atomic<float>(L, N, M, K, R, offsets);
+    if (type == 4) time2 = test_no_atomic<double>(L, N, M, K, R, offsets);
+    if (type == 5)
+      time2 = test_no_atomic<Kokkos::complex<double> >(L, N, M, K, R, offsets);
```

Comment:
@crtrott this should go to the same line of the IF statement, shouldn't it?

Reply:
My guess is this is due to line length issues. At some point you have to get it on a new line. So I think this is fine.